packages/react-storage/src/components/StorageBrowser/Views/LocationsListView.tsx

```diff
@@ -17,5 +19,22 @@ export default function LocationsListView(): JSX.Element {
     ? null
     : data.locations.map(({ name }) => <p key={name}>{name}</p>);
 
-  return <>{!hasLocations && isLoading ? 'loading...' : listLocations}</>;
+  return (
+    <>
+      <div className={`${CLASS_BASE}__header`}>
```

Comment:
Need to determine where these divs with classNames actually need to live, for now they are just here for styling purposes.